package.json

```
@@ -40,8 +42,10 @@
     "@vitejs/plugin-vue": "2.2.4",
     "@vue/eslint-config-typescript": "10.0.0",
     "@vue/test-utils": "2.0.0-rc.18",
+    "cypress": "^9.5.1",
```

always use fixed version:

```suggestion
    "cypress": "9.5.1",
```